### ISSUE TITLE
xbps: More verbose logging

### ISSUE THREAD

**Issue by camoz** [void-linux/xbps#450]
I think it would be helpful if xbps also logged:

- The exact command that is being run
- Maybe a short notice about start and end of the transaction
- messages from `INSTALL.msg` files that are displayed during interactive installation
- notes about config files that are not overwritten, like during interactive installation
- which triggers/hooks are being run + the output of them

The purpose of this is to better be able to understand in retrospect what was done to a system, e.g. when the output of an interactive command was too long to be saved in scrollback.

**Comment by Duncaen** [comment]
> * The exact command that is being run

I don't think this is necessary, logging what is being done would be more useful than the raw flags.

> * messages from INSTALL.msg files that are displayed during interactive installation

I'm really not a fan of `INSTALL.msg` since they are currently used for unimportant things like documentation and not just important messages and they are shown on each following update.

> * which triggers/hooks are being run + the output of them

We can't know, all we know is that we execute the INSTALL or REMOVE script and its exit status, those scripts invoke the triggers.
